src/impl/chassis/controller/chassisControllerBuilder.cpp

```diff
@@ -413,6 +413,12 @@ ChassisControllerBuilder::buildDOCC(std::shared_ptr<ChassisController> chassisCo
 }
 
 std::shared_ptr<ChassisControllerPID> ChassisControllerBuilder::buildCCPID() {
+  auto ccGearset = gearset;
+  auto ccDriveScales = driveScales;
+  if(differentOdomScales) {
+    ccDriveScales = odomScales;
+    ccGearset = AbstractMotor::gearset::green;
```

Comment:
Why hard-code green?

Reply:
Yeah, that may not be necessary. will test

Reply:
@Octogonapus - During physical testing I ran into issues with an externally geared chassis which I've also resolved in this PR. Feedback?